modules/axtask/src/wait_queue.rs

```diff
@@ -29,57 +38,41 @@ use crate::{current_run_queue, select_run_queue, AxTaskRef, CurrentTask};
 /// assert_eq!(VALUE.load(Ordering::Relaxed), 1);
 /// ```
 pub struct WaitQueue {
-    queue: SpinNoIrq<VecDeque<AxTaskRef>>,
+    queue: SpinNoIrq<WaitTaskList>,
```

Comment:
Why do we wrap `WaitTaskList` with `WaitQueue`,
instead of use `list: SpinNoIrq<List<Node>>` here directly

Reply:
OK goog idea , it have done